.github/workflows/connector_ops_ci.yml

```diff
@@ -21,3 +22,35 @@ jobs:
         run: pip install --quiet -e ./tools/ci_connector_ops
       - name: Check test strictness level
         run: check-test-strictness-level
+  requires-connector-team-review:
+    name: "Check if a review is required from Connector teams"
+    runs-on: ubuntu-latest
+
+    # GitHub should provide a "pull_request_review_target", but they don't and
+    # the action will fail if run on a forked PR.
+    if: github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
```

Comment:
I've done this kind of thing in the past:

```
if: ${{ github.repository == 'airbytehq/airbyte' }}
```

Reply:
👍 Thanks!